scripts/e2e_lambda_deployment.py

```diff
@@ -0,0 +1,141 @@
+import shutil
+import subprocess
+import logging
+import requests
+import json
+
+from sklearn import svm, datasets
+from bentoml import BentoService, load, api, env, artifacts
+from bentoml.artifact import PickleArtifact
+from bentoml.handlers import DataframeHandler
+from bentoml.proto.repository_pb2 import DangerouslyDeleteBentoRequest
+from bentoml.yatai import get_yatai_service
+
+
+logger = logging.getLogger(__name__)
+
+
+@artifacts([PickleArtifact('clf')])
+@env(pip_dependencies=['scikit-learn'])
+class IrisClassifier(BentoService):
+    @api(DataframeHandler)
+    def predict(self, df):
+        return self.artifacts.clf.predict(df)
+
+
+if __name__ == '__main__':
+    print('E2E DEPLOYMENT TEST FOR LAMBDA:::: Training iris classifier')
```

Comment:
use logger.info?, change logger to logger = logging.getLogger("bentoml.test")
 ?